plugins/org.locationtech.udig.catalog.db2/src/org/locationtech/udig/catalog/db2/internal/messages.properties

```diff
@@ -15,5 +15,7 @@ DB2WizardPage_installDrivers=Install DB2 drivers from client
 DB2WizardPage_connectionTask=Attempting to make database connection
 DB2WizardPage_warning=An error will occur because the DB2 drivers were not loaded.  Please restart application
 DB2WizardPage_title=DB2 Wizard
+DB2WizardPage_button_lookup_text=Get schemas
```

Comment:
Its easier than expected: Just build with option **_-Pdocs_** (Details [here](https://github.com/locationtech/udig-platform#option--command-line-build)). Output html you can find down below doc/target
HTH